### DISCUSSION TITLE
Upgrading to the latest Tailwind

### DISCUSSION THREAD

**Discussion by FrozenHearth** [EddieHubCommunity/LinkFree#5685] [Ideas]
We don't need to use the line-clamp plugin anymore! 🎉

[Tailwind v3.3](https://twitter.com/adamwathan/status/1640729559606734848?s=46&t=Rud_3ojydDYKRYHyMOoN1A)

**Comment by eddiejaoude** [comment]
I think this has been done but if not please raise an issue
